ansible/monitoring.yml

```diff
@@ -44,10 +44,14 @@
     datadog_api_key: "{{ vault_datadog_api_key|default('DATADOG_API_KEY') }}"
     datadog_agent6: true
     datadog_config:
+      log_level: warning
```

Comment:
I do not easily follow where this is documented or what it means. Does it mean to report severity warning or higher - implying that any input shall be structurally known to have a certain severity? Or does it plainly **add** warning to any message? (Even if informational.)

Reply:
This is datadog configuration, it lowers (datadog) logging severity to warning from info (the default). This is to reduce useless logs that we start feeding to the logs collector (still datadog in our case).